shell/platform/embedder/embedder_external_view_embedder.cc

```diff
@@ -95,24 +95,14 @@ void EmbedderExternalViewEmbedder::BeginFrame(SkISize frame_size,
     }
   }
 
-  // If there is no root render target, create one now. This will be accessed by
-  // the rasterizer before the submit call layer to access the surface surface
-  // canvas.
+  // If there is no root render target, create one now.
+  // TODO(chinmaygarde): This can now be moved to be later in the submit call.
```

Comment:
nit: If it's something you think we should do let's have a tracking bug and change the language from "can" to "should"

Reply:
Done and filed https://github.com/flutter/flutter/issues/43778